Core/src/org/sleuthkit/autopsy/datamodel/accounts/Accounts.java

```diff
@@ -249,12 +255,69 @@ public String getItemType() {
             return getClass().getName();
         }
     }
+    
+    /**
+     * Tracks the account types and the number of account types found.
+     */
+    private class AccountTypeResults {
+        private final Map<String, Long> counts = new HashMap<>();
+        
+        AccountTypeResults() {
+            update();
+        }
+        
+        /**
+         * Given the type name of the Account.Type, provides the count of those type.
+         * @param accountType   The type name of the Account.Type.
+         * @return              The number of results found for the given account type.
+         */
+        Long getCount(String accountType) {
+            return counts.get(accountType);
+        }
+        
+        /**
+         * Retrieves an alphabetically organized list of all the account types.
+         * @return      An alphabetically organized list of all the account types.
+         */
+        List<String> getTypes() {
+            List<String> types = new ArrayList(counts.keySet());
+            Collections.sort(types);
```

Comment:
This method causes several build warning.

Reply:
Ok.  I believed this is fixed.